book/src/concepts/component.md

```diff
@@ -16,19 +16,35 @@ In this example, the bottle is the entity, and the properties are components.
 
 Creating a component is easy.
 
-You declare a struct:
+You declare the relevant structs:
 
 ```rust,ignore
-pub struct MyComponent {
-    some_property: String,
+enum Shape {
```

Comment:
Why not just impl Component for enum Shape?

Reply:
When you're right, you're right. Good call.

Reply:
Actually wow, I can't believe I didn't notice this won't compile. Sloppy of me.